pkg/buildpack/multi_architecture_helper.go

```diff
@@ -39,11 +39,14 @@ func (m *MultiArchConfig) Targets() []dist.Target {
 // CopyConfigFiles will, given a base directory (which is expected to be the root folder of a single buildpack),
 // copy the buildpack.toml file from the base directory into the corresponding platform root folder for each target.
 // It will return an array with all the platform root folders where the buildpack.toml file was copied.
-func (m *MultiArchConfig) CopyConfigFiles(baseDir string) ([]string, error) {
+func (m *MultiArchConfig) CopyConfigFiles(baseDir string, buildpackType string) ([]string, error) {
 	var filesToClean []string
+	if buildpackType == "" {
+		buildpackType = "buildpack"
```

Comment:
```suggestion
		buildpackType =  KindBuildpack
```

I think we can reuse some constants we already have defined [here](https://github.com/buildpacks/pack/blob/main/pkg/buildpack/buildpack.go#L21-L24)